cli/Cargo.toml

```diff
@@ -105,6 +105,7 @@ tokio-util = "=0.7.2"
 tower-lsp = "=0.17.0"
 twox-hash = "=1.6.2"
 typed-arena = "2.0.1"
+update-informer = { version = "0.5.0", default_features = false, features = ["github"] }
```

Comment:
Doing this is pretty trivial so I'd rather we not use a dependency for this. We also want to check stuff like canary builds.

There is also another open PR #15543.